service/lib/dinstaller/dbus/service_runner.rb

```diff
@@ -62,28 +58,21 @@ def run
 
       attr_reader :name, :logger
 
-      # Set up the manager service
-      #
-      # @param logger [Logger] Service logger
-      # @return [DInstaller::DBus::Service] Manager service
-      def setup_manager(logger)
-        manager = DInstaller::Manager.new(logger)
-        manager.setup
-        service = DInstaller::DBus::Service.new(manager, logger)
-        manager.probe # probe after export, so we can report on DBus
-        manager.progress.on_change { service.dispatch } # make single thread more responsive
-        service
+      # Configuration
+      def config
+        Config.load unless Config.current
+        Config.current
```

Comment:
maybe we should add that load to current method? So it lazy loads and user of config do not need to care?

Reply:
It makes sense. But I would like to move the decision to #189. I will make a note there.